Apps/AdminWebClient/src/ClientApp/src/components/core/CommunicationTable.vue

```diff
@@ -276,12 +280,12 @@ export default class CommunicationTable extends Vue {
             .then((banners: Communication[]) => {
                 this.parseComms(banners);
             })
-            .catch((err) => {
+            .catch(err => {
```

Comment:
I am in favour of this style change, though it doesn't matter much to me either way.

Reply:
missing the prettier library caused this too